lang/LangSource/GC.h

```diff
@@ -82,51 +84,47 @@ class PyrGC
 
 	MALLOC PyrObject* NewFinalizer(ObjFuncPtr finalizeFunc, PyrObject *inObject, bool inCollect);
 
-	bool IsBlack(PyrObjectHdr* inObj)  { return inObj->gc_color == mBlackColor; }
-	bool IsWhite(PyrObjectHdr* inObj)  { return inObj->gc_color == mWhiteColor; }
-	bool IsGrey(PyrObjectHdr* inObj)   { return inObj->gc_color == mGreyColor; }
-	static bool IsMarker(PyrObjectHdr* inObj) { return inObj->gc_color == obj_gcmarker; }
-	bool IsFree(PyrObjectHdr* inObj)   { return (!(IsMarker(inObj) ||
+	BOOST_FORCEINLINE bool IsBlack(PyrObjectHdr* inObj)  { return inObj->gc_color == mBlackColor; }
```

Comment:
If anybody's wondering:

// BOOST_FORCEINLINE ---------------------------------------------//
// Macro to use in place of 'inline' to force a function to be inline
#if !defined(BOOST_FORCEINLINE)
#  if defined(_MSC_VER)
#    define BOOST_FORCEINLINE __forceinline
#  elif defined(__GNUC__) && __GNUC__ > 3
     // Clang also defines __GNUC__ (as 4)
#    define BOOST_FORCEINLINE inline __attribute__ ((__always_inline__))
#  else
#    define BOOST_FORCEINLINE inline
#  endif
#endif